.github/workflows/kind_host.yml

```diff
@@ -0,0 +1,140 @@
+name: Framework KinD hosted mode Workflow
+
+on:
+  workflow_call:
+
+env:
+  RELEASE_BRANCH: ${{ github.event.pull_request.base.ref || github.ref_name }}
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  tests:
+    name: Hosted mode Tests # Part of the check name, be careful when changing.
+
+    runs-on: ubuntu-latest
+    env:
+      REGISTRY: localhost:5000
+      KIND: latest
+      DEPLOYONHUB: 'false'
+    strategy:
+      # Don't skip all tests if one fails - others may still have useful info
+      fail-fast: false
+
+    steps:
+    - name: Checkout Component Repository
+      # Checkout a specific component when called from another repository
+      if: ${{ github.event.repository.name != 'governance-policy-framework' }}
+      uses: actions/checkout@v3
+      with:
+        # `repository` is inferred as the "caller" repository 
+        path: component
+        # `ref` is inferred as the new commit in the "caller" repository
+
+    - name: Checkout Policy Framework
+      # Checkout a stable branch when called from another repository
+      if: ${{ github.event.repository.name != 'governance-policy-framework' }}
+      uses: actions/checkout@v3
+      with:
+        repository: stolostron/governance-policy-framework
+        path: framework
+        ref: ${{ env.RELEASE_BRANCH }} # like main or release-*
+
+    - name: Checkout Policy Framework
+      # Checkout like "usual" when called from this repository
+      if: ${{ github.event.repository.name == 'governance-policy-framework' }}
+      uses: actions/checkout@v3
+      with:
+        # `repository` is inferred as the "caller" repository 
+        path: framework
+        # `ref` is inferred as the new commit
+
+    - name: Set up Go
+      uses: actions/setup-go@v3
+      with:
+        go-version-file: framework/go.mod
+
+    - name: Bootstrap the KinD Cluster for hosted mode
+      working-directory: framework
+      run: |
+        echo "::group::make e2e-dependencies"
+        make e2e-dependencies
+        echo "::endgroup::"
+
+        echo "::group::make setup hosted mode"
+        make bootstrap-hosted
+        echo "::endgroup::"
+        echo "::group::Saving kubeconfig paths for use in other steps"
+    
+        echo "MANAGED_KUBECONFIG=$(pwd)/governance-policy-addon-controller/policy-addon-ctrl2.kubeconfig" >> $GITHUB_ENV
+        echo "HUB_KUBECONFIG=$(pwd)/governance-policy-addon-controller/policy-addon-ctrl1.kubeconfig" >> $GITHUB_ENV
+        echo "HUB_INTERNAL_KUBECONFIG=$(pwd)/governance-policy-addon-controller/policy-addon-ctrl1.kubeconfig-internal" >> $GITHUB_ENV
```

Comment:
Since these are copied over to the root framework directory in the Makefile, it seems it might be clearer to use the kubeconfigs at the root of the repo rather than the ones from the addon controller?